tests/dummy/app/routes/new-review.js

```diff
@@ -1,6 +1,10 @@
 import Ember from 'ember';
 import gql from 'graphql-tag';
 
+// working around browserify limitations:
+import 'npm:common-tags';
```

Comment:
I can't for the life of me understand why this would be necessary to making the tests pass. Doesn't look like it's actually used anywhere?

As an aside, looks like I forgot to remove it from the dependencies too.